lua/examples/example_of_dynamic_config_request.lua

```diff
@@ -146,4 +146,21 @@ if ret < 0 then
 	return "gk: failed to flush the flow table\n"
 end
 
+ret = dylib.c.gk_load_bpf_flow_handler(dyc.gk, 255, "bpf/granted.bpf", true)
+if ret < 0 then
```

Comment:
All changes in file `lua/examples/example_of_dynamic_config_request.lua` should go to patch `bpf: add the first BPF programs` since the BPF program `"bpf/granted.bpf"`  is only available there.